root/report/DuplicateArtists.js

```diff
@@ -27,42 +25,34 @@ const DuplicateArtists = ({
   generated,
   items,
   pager,
-}: ReportDataT<ReportArtistT>): React.Element<typeof Layout> => {
+}: ReportDataT<ReportArtistT>): React.Element<typeof ReportLayout> => {
   let currentKey = '';
   let lastKey = '';
 
   return (
-    <Layout $c={$c} fullWidth title={l('Possibly duplicate artists')}>
-      <h1>{l('Possibly duplicate artists')}</h1>
-
-      <ul>
-        <li>
-          {exp.l(
-            `This report aims to identify artists with very similar names.
-             If two artists are actually the same, please merge them
-             (remember to {how_to_write_edit_notes|write an edit note}
-             and give your proof). If they\'re different, add
-             {disambiguation_comment|disambiguation comments} to them
-             (and once a group of similarly named artists have
-             disambiguation comments, they will stop appearing here).`,
-            {
-              disambiguation_comment: '/doc/Disambiguation_Comment',
-              how_to_write_edit_notes: '/doc/How_to_Write_Edit_Notes',
-            },
-          )}
-        </li>
-        <li>
-          {texp.l('Total duplicate groups: {count}',
-                  {count: pager.total_entries})}
-        </li>
-        <li>
-          {texp.l('Generated on {date}',
-                  {date: formatUserDate($c, generated)})}
-        </li>
-
-        {canBeFiltered ? <FilterLink $c={$c} filtered={filtered} /> : null}
-      </ul>
-
+    <ReportLayout
+      $c={$c}
+      canBeFiltered={canBeFiltered}
+      countText={l('Total duplicate groups: {count}')}
```

Comment:
This is the only case where a separate countText is needed, but since I guess it does count groups and not artists, it's probably still needed to do it like this.